### DISCUSSION TITLE
benchmark chart

### DISCUSSION THREAD

**Discussion by juancarlospaco** [juancarlospaco/faster-than-csv#16] [General]
Hey @SekouDiaoNlp do you know a simple way to make https://github.com/juancarlospaco/faster-than-csv/blob/master/benchmark.py
generate some chart like PNG or SVG ?

For the readme, something more visual than the markdown table, if it is not too complicated. :smile:

**Comment by SekouDiaoNlp** [comment]
Hi @juancarlospaco , what's up?

Yes I can quic kly whip-up some charts in `svg`/`png `using `matplotlib`.

A bar chart with the standard library `readcsv()` at 100 and the rest relative to that.

Should I make it bigger bar is slower or bigger bar is faster.

We can also improve the benchmark by using different sizes of `csv `sizes in row and column numbers to observe the influence of input size on relative performance.

@SekouDiaoNlp

**Reply by juancarlospaco** [reply]
Whatever you like about the chart design.

Yes theres a big stresstests at https://github.com/juancarlospaco/faster-than-csv/tree/master/examples/stresstest

**Reply by SekouDiaoNlp** [reply]
Ok, I will whip out something tomorrow, maybe different designs and we can choose from there.

**Comment by juancarlospaco** [comment]
This is done!. :smile:
